libr/core/disasm.c

```diff
@@ -4782,7 +4782,11 @@ static char *ds_sub_jumps(RDisasmState *ds, char *str) {
 	if (fcn) {
 		name = fcn->name;
 	} else if (f) {
-		RBinReloc *rel = r_core_getreloc (ds->core, addr, ds->analop.size);
+		RBinReloc *rel;
+		rel = r_core_getreloc (ds->core, ds->analop.addr, ds->analop.size);
```

Comment:
umf, why addr != ds->analop.addr ?

Reply:
ds->analop.addr is the address of the current instruction
addr is where the instruction is telling you to jump

Reply:
confusing varnames then :) ok to merge